src/apps/chifra/pkg/types/types_trace.go

```diff
@@ -286,6 +310,9 @@ func GetTracesByTransactionHash(chain string, txHash string) ([]SimpleTrace, err
 				Action:           &action,
 				Result:           result,
 			}
+			if transaction != nil {
```

Comment:
It's very, very unclear to me why the RPC doesn't do this already. It's so annoying.